src/python/WMComponent/WorkflowUpdater/SiteListPoller.py

```diff
@@ -127,7 +127,8 @@ def algorithm(self, parameters=None):
                 self.logger.info("  siteBlackList %s => %s", wmaBlackList, siteBlackList)
                 try:
                     # update local WorkQueue first
-                    self.localWQ.updateSiteLists(wflow, siteWhiteList, siteBlackList)
+                    params = {'SiteWhiteList': siteWhiteList, 'SiteBlackList': siteBlackList}
```

Comment:
And I just noticed that this - and the previous implementation - were setting the wrong workqueue element parameters. Please note the case sensitivity here:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WorkQueue/DataStructs/WorkQueueElement.py#L73-L74

and update this code.

Reply:
ok, done.